CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ cmake_minimum_required(VERSION 3.12...3.28)
 
 set(ROOT_PROJECT_NAME ICUB)
 project(${ROOT_PROJECT_NAME} LANGUAGES C CXX
-                             VERSION 2.7.1)
+                             VERSION 2.7.2)
```

Comment:
Unless strictly required please leave it unchanged

Reply:
Oks, I'll leave the version to 2.7.1 then. So, which versioning should I care about. Just icub-firmware-shared?

Reply:
> So, which versioning should I care about. Just icub-firmware-shared and update conf/iCubFindDependencies.cmake accordingly and that's all?

Yep, thanks!